PIPReader/src/test/java/it/cnr/iit/ucs/properties/UCFPipProperties.java

```diff
@@ -61,4 +58,19 @@ public void setAttributes( List<Map<String, String>> attributes ) {
         return null;
     }
 
+    @Override
+    public String getJournalPath() {
+        return journalDir;
+    }
+
+    @Override
+    public String getJournalProtocol() {
```

Comment:
we could put this in the base property class.
each component could have a journal configured